### PR TITLE
Add trademark search, update db, send email django management command

### LINKED ISSUE #1025: Add trademark search

### PR DESCRIPTION
**Pull request by drvcodenta**
fixes: #1025

### CODE REVIEW

**Comment by drvcodenta** [comment]
resolved

**Review by DonnieBLT — CHANGES_REQUESTED** [review]
two changes please

File: website/models.py [hunk 1]
```diff
@@ -123,6 +123,7 @@ class Company(models.Model):
     is_active = models.BooleanField(default=False)
     tags = models.ManyToManyField(Tag, blank=True)
     integrations = models.ManyToManyField(Integration, related_name="companies")
+    trademark = models.IntegerField(default=0)
```
[the remark this answers is not in the document]
Comment (drvcodenta) [reply website/models.py:126]: ok

[the remark this answers is not in the document]
Comment (drvcodenta) [reply website/models.py:126]: Do I need to create schedule, that checks the value every week automatically?

**Pull request closed without merging**